extensions/amp-access/0.1/amp-access.js

```diff
@@ -359,17 +359,17 @@ export class AccessService {
    */
   runAuthorization_(opt_disableFallback) {
     if (this.config_.type == AccessType.OTHER) {
-      log.fine(TAG, 'Ignore authorization due to type=other');
+      user.fine(TAG, 'Ignore authorization due to type=other');
       this.firstAuthorizationResolver_();
       return Promise.resolve();
     }
 
-    log.fine(TAG, 'Start authorization via ', this.config_.authorization);
+    dev.fine(TAG, 'Start authorization via ', this.config_.authorization);
     this.toggleTopClass_('amp-access-loading', true);
     const urlPromise = this.buildUrl_(
         this.config_.authorization, /* useAuthData */ false);
     const promise = urlPromise.then(url => {
-      log.fine(TAG, 'Authorization URL: ', url);
+      devlog.fine(TAG, 'Authorization URL: ', url);
```

Comment:
Correct. But I will simply revert this file - I'd rather do replacements globally through the whole codebase. It's here for now as a demo.